docs/migration-guides/1.15-1.16.md

```diff
@@ -4,6 +4,8 @@
 
 Kyma 1.16 comes with a new Istio version: 1.5.10.
 
+It introduces new Custom Resource for Istio installation. If you use any overrides provided by Helm or IstioControlPlane migrate them to the new [IstioOperator](https://istio.io/v1.5/docs/reference/config/istio.operator.v1alpha1/). See [an example of IstioOperator](https://kyma-project.io/docs/1.16/components/service-mesh/#configuration-istio-custom-configuration) for more information.
```

Comment:
The "It" in the sentence can refer to Kyma or Istio. Please make it clear, maybe like that: "Istio 1.5.10 introduces new..." A bit repetitive, perhaps TW will help with that?

Reply:
I think both introduce this ;)